cves/kernel/CVE-2012-6545.yml

```diff
@@ -161,10 +154,14 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    It seems this was a minor memory leak discovered and fixed by a developer. 
+    The only written history on this are the git commits, and this email sent 
+    by the developer who fixed it: https://www.openwall.com/lists/oss-security/2013/03/05/13. 
+    It was small enough to go unreported for some years.
```

Comment:
Consider adding the exact amount of years between introduction and discovery.

Reply:
It was only a year after looking at the dates.